solutions/devsprint-cecilia-saraiva-2/FinanceAppTests/Strings+ExtensionsTests.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  Strings+ExtensionsTests.swift
+//  FinanceAppTests
+//
+//  Created by Raline Maria da Silva on 16/08/22.
+//
+
+import XCTest
+@testable import FinanceApp
+
+final class Strings_ExtensionsTests: XCTestCase {
```

Comment:
[sugestao] talvez possamos tirar o `_` e ter as duas palavras juntas.
```suggestion
final class StringsExtensionsTests: XCTestCase {
```

Reply:
feito